about/index.md

```diff
@@ -4,8 +4,63 @@
 ## Banner code and credit
 This [banner](https://raw.githubusercontent.com/pymc-devs/pymc-data-umbrella/main/banner.png) is generated from [this code](https://raw.githubusercontent.com/pymc-devs/pymc-data-umbrella/main/banner.py), the code in this link is a trivial customization of the [original code](https://github.com/pymc-devs/pymcon/blob/gh-pages/assets/make_trajectories.py) by [Colin Caroll](https://colindcarroll.com) who designed a [similar banner for pymcon'20](https://pymcon.com/), Colin is amazing at visualization stuff and even has a couple of [talks](https://github.com/ColCarroll/yourplotlib) about it!!
 
+## Organizers
+
+::::{grid} 2 2 4 4
```

Comment:
changes this to grids for two main reasons. First is to make it mobile friendly, we are defining these `2 2 4 4` which means 2 columns in small and xs screens, 4 columns in medium and large screens. Moreover, the whole "card" is clickable now instead of only the name. I tried to keep the style as similar as possible to the original tables.

Reply:
makes sense, i was thinking of the bad version on mobile as well, thanks!